app/models/task.py

```diff
@@ -1,5 +1,22 @@
 from app import db
-
+from datetime import datetime 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
```

Comment:
Same comment as in the Goal class, prefer that the primary key attribute is called `id`